djstripe/views.py

```diff
@@ -252,6 +253,17 @@ class CancelSubscriptionView(LoginRequiredMixin, SubscriptionMixin, FormView):
     template_name = "djstripe/cancel_subscription.html"
     form_class = CancelSubscriptionForm
     success_url = reverse_lazy("djstripe:account")
+    redirect_url = reverse_lazy("home")
+
+    # messages
+    subscription_cancel_msg = "Your subscription is now cancelled."
+    subscription_status_msg = "Your subscription status is now '{status}' until '{period_end}'"
+
+    def get_redirect_url(self):
```

Comment:
Solid. Instead of short-circuiting, please add an else.

Also, add a newline before the conditional and make sure to add a docstring.